marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx

```diff
@@ -30,6 +30,87 @@ describe('Entity Modeling', () => {
       cy.resetTestUser();
   });
 
+  it('can add new properties to existing Entity, and revert the entity', () => {
+    // Adding property to Order entity
+    entityTypeTable.expandEntityRow(0);
+    propertyTable.getAddPropertyButton('Order').click();
+
+    propertyModal.newPropertyName('orderID');
+    propertyModal.openPropertyDropdown();
+    propertyModal.getTypeFromDropdown('string').click();
+    propertyModal.getNoRadio('identifier').click();
+    propertyModal.getYesRadio('multiple').click();
+    propertyModal.getYesRadio('pii').click();
+    propertyModal.clickCheckbox('wildcard');
+    propertyModal.getSubmitButton().click();
+
+    propertyTable.getMultipleIcon('orderID').should('exist');
+    propertyTable.getPiiIcon('orderID').should('exist');
+    propertyTable.getWildcardIcon('orderID').should('exist');
+
+
+    entityTypeTable.getRevertEntityIcon('Order').should('exist');
+    entityTypeTable.getRevertEntityIcon('Order').click();
+    confirmationModal.getYesButton(ConfirmationType.RevertEntity).click();
+    confirmationModal.getRevertEntityText().should('exist');
+    confirmationModal.getRevertEntityText().should('not.exist');
+
+    propertyTable.getMultipleIcon('orderID').should('not.exist');
+    propertyTable.getPiiIcon('orderID').should('not.exist');
+    propertyTable.getWildcardIcon('orderID').should('not.exist');
+  });
+
+  it('can add new properties to existing Entities, and revert all entities', () => {
```

Comment:
These actions can be in front of the save all entity test.

Reply:
Updated tests.